test/setup-deployment.sh

```diff
@@ -40,9 +40,6 @@ DEPLOY=(
 )
 echo "INFO: deploying from ${DEPLOYMENT_DIRECTORY}/${TEST_DEVICEMODE}${deployment_suffix}"
 
-# Set up TLS secrets in the TEST_DRIVER_NAMESPACE.
-PATH="${REPO_DIRECTORY}/_work/bin:$PATH" KUBECTL="${KUBECTL}" ${TEST_DIRECTORY}/setup-ca-kubernetes.sh
-
```

Comment:
This breaks skew testing.

Reply:
The version skew test does not use version specific `test/setup-deployment.sh` ? If yes, the degrading version supposed to create these secrets with that specific deployment script?

Reply:
I might have confused this with some other place. Can you test that skew testing still works locally or submit your branch to the main repo so that the CI can test it?

Reply:
I think I remember now: some skew tests mimick a partial downgrade by replacing just the controller. That then uses the installation as prepared by the master setup-deployment.sh together with an older image that needs these secrets.

Reply:
I made changes to the scripts to generate so that they only 'webhook' secrets. And the local "skew controller" tests got succeeded. I pushed to this branch to main repo.